### PR TITLE
[EDU-2735] refactor: Update Next.js Static Boilerplate guide

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue

[EDU-2735]

### Changes

Update Next.js Static Boilerplate guide, adding static exports section.

### Additional links
n/a.

[EDU-2735]: https://aziontech.atlassian.net/browse/EDU-2735?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/guias/marketplace/templates/next-static-boilerplate/next-static-boilerplate.mdx [hunk 1]
```diff
@@ -18,6 +18,41 @@ Este template usa o Next.js versão `13.5.3`.
 
 ---
 
+## Sobre static exports
+
+O núcleo do Next.js oferece suporte a exportações estáticas. Isso permite que você inicie seu projeto como um site estático ou uma SPA e, em seguida, adicione recursos que requerem um servidor. Ao usar esses recursos mais avançados, Next.js gera um arquivo HTML por rota, dividindo um SPA estrito em arquivos HTML individuais, e seu projeto pode ser implantado e hospedado em qualquer servidor web que possa servir ativos estáticos HTML/CSS/JS.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/next-static-boilerplate/next-static-boilerplate.mdx:23]:
```suggestion
O núcleo do Next.js oferece suporte a exportações estáticas. Isso permite que você inicie seu projeto como um site estático ou uma SPA e, em seguida, adicione recursos que requerem um servidor. Ao usar esses recursos mais avançados, o Next.js gera um arquivo HTML por rota, dividindo um SPA estrito em arquivos HTML individuais, e seu projeto pode ser implantado e hospedado em qualquer servidor web que possa servir ativos estáticos HTML/CSS/JS.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/next-static-boilerplate/next-static-boilerplate.mdx [hunk 2]
```diff
@@ -18,6 +18,41 @@ Este template usa o Next.js versão `13.5.3`.
...
+- Componentes do servidor
+- Componentes do cliente
+- Otimização de imagem
+- Otimização de imagem (através de `next/image` pode ser usada com uma exportação estática definindo um carregador de imagem personalizado em `next.config.js`)
+- Manipuladores de rota
+- APIs do navegador
+
+### Recursos não suportados
+
+Esses recursos exigem um servidor Node.js ou uma lógica dinâmica que não pode ser computada durante o processo de compilação e não tem suporte:
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/next-static-boilerplate/next-static-boilerplate.mdx:40]:
```suggestion
Esses recursos exigem um servidor Node.js ou uma lógica dinâmica que não pode ser computada durante o processo de compilação e não têm suporte:
```

Comment (hannah-sk) [reply src/content/docs/pt-br/pages/guias/marketplace/templates/next-static-boilerplate/next-static-boilerplate.mdx:40]: acento supondo que está se referindo a "recursos"

**Review by hannah-sk — APPROVED** [review]
deixei 2 sugestões

File: src/content/docs/en/pages/guides/marketplace/templates/nextjs-static-boilerplate/nextjs-static-boilerplate.mdx [hunk 3]
```diff
@@ -18,12 +18,47 @@ This template uses Next.js version `18.2.0`.
 
 ---
 
+## About static exports
+
+The core of Next.js supports static exports. This allows you to initiate your project either as a static site or a SPA and then add features that require a server. By using these more advanced features, Next.js generates an HTML file per route, breaking a strict SPA into individual HTML files, and your project can be deployed and hosted on any web server that can serve HTML/CSS/JS static assets.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/guides/marketplace/templates/nextjs-static-boilerplate/nextjs-static-boilerplate.mdx:23]:
```suggestion
The core of Next.js supports static exports. This allows you to initiate your project either as a static site or a SPA and then add features that require a server. By using these advanced features, Next.js generates one HTML file per route, breaking a strict SPA into individual HTML files, and your project can be deployed and hosted on any web server that can serve HTML/CSS/JS static assets.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/next-static-boilerplate/next-static-boilerplate.mdx [hunk 4]
```diff
@@ -18,7 +18,42 @@ Este template usa o Next.js versão `13.5.3`.
...
+- Otimização de imagem (através de `next/image` pode ser usada com uma exportação estática definindo um carregador de imagem personalizado em `next.config.js`)
+- Manipuladores de rota
+- APIs do navegador
+
+### Recursos não suportados
+
+Esses recursos exigem um servidor Node.js ou uma lógica dinâmica que não pode ser computada durante o processo de compilação e não têm suporte:
+
+- Rotas dinâmicas com dynamicParams: true
+- Rotas dinâmicas sem generateStaticParams()
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/next-static-boilerplate/next-static-boilerplate.mdx:43]:
```suggestion
- Rotas dinâmicas com `dynamicParams: true`
- Rotas dinâmicas sem `generateStaticParams()`
```

**Review by bru-andrade — APPROVED** [review]
Muito bommm :D

**Pull request merged**